FabricTestExample/package.json

```diff
@@ -16,31 +16,31 @@
     "@react-navigation/native-stack": "^6.9.13",
     "@react-navigation/stack": "^6.3.17",
     "react": "18.2.0",
-    "react-native": "0.72.4",
+    "react-native": "0.73.0-rc.3",
     "react-native-gesture-handler": "^2.12.1",
-    "react-native-reanimated": "3.4.2",
-    "react-native-safe-area-context": "^4.7.1",
+    "react-native-reanimated": "software-mansion/react-native-reanimated#c5913665dbed5dfbcf46ff84b0b3feef1196a8ac",
+    "react-native-safe-area-context": "th3rdwave/react-native-safe-area-context#2917160fa30635751c42734f7fc40e7273ea0fe7",
```

Comment:
Support for 0.73 in RNSAC has not been merged to main yet, thus we are sticking to the commit.
Yeah, since Reanimated 3.6.0 was released I can bump that.

Reply:
https://github.com/software-mansion/react-native-screens/pull/1956/commits/e4792e4ef9c3e40707eaa1cfe2eec9523710a65f